### DISCUSSION TITLE
HTML / markdown widget

### DISCUSSION THREAD

**Discussion by Acanis87** [TomSchimansky/CustomTkinter#2514] [Ideas]
Hello,

I'm pretty new to Python and have discovered CustomTkinter, which I'm really enjoying so far! It's intuitive, clean, and just nice to use.

For my first project, I'd like to create a small info modal with some information about the program. I've created a markdown file with a description and am now trying to find a way to display it.

I did find a workaround using "from tkhtmlview import HTMLLabel," but it wasn't ideal.

I would really appreciate a way to display formatted markdown. Perhaps a widget that can display HTML – in conjunction with the "markdown" package – would be the most intuitive solution.

**Comment by dipeshSam** [comment]
Tkinter is a low-level GUI building tool hence advanced engines like HTML or Markdown Renderers are not available as built-in. But you can write one by your own.

Helpful links: https://github.com/andrewlin12/markdown2png

Regards.

**Reply by Acanis87** [reply]
How does that link help? Oo
Its a 14y old repo and all he does, is using:
`    img = md2png.md2png(f.read(), [(0, 0, width)], {
        "font_size": 16,
        "code_font_size": 18,
    })`

**Comment by Acanis87** [comment]
Iam just using HTML and tkhtmlview for now. But its not really working **too** well (doesnt use the styles of customtkinter) and using markdown would be easier to maintain. I found some markdown-viewer and maybe could have used some of that code, but as beginner thats quite a task ^^' 

For my actual case, Iam fine with using html. 

_Still I would be happy to see a new widget for customtkinter in the future :)_

**Comment by lukagouvea** [comment]
Hey, check [https://github.com/lukagouvea/CTkMarkdown](https://github.com/lukagouvea/CTkMarkdown) although it is not a complete Markdown renderer it support the basic syntax and fit seamlessly with customtkinter

**Reply by dipeshSam** [reply]
Appreciable!
